tests/testthat/test-endpoints.R

```diff
@@ -1,24 +1,39 @@
 context("endpoints")
 
-test_that("endpoint_validate_baseline correctly validates data", {
+validate_test_that("endpoint_validate_input correctly validates data", {
```

Comment:
I had a question about this. So we're only doing schema validation when an environmental variable is set. What is the best way to make this work? Is there a way to set the env var for all tests from one place?

Reply:
I would set the environment variable machine-by-machine: so in your .Renviron, and in the .travis.yml